js/build/shortcode-ui.js

```diff
@@ -1768,7 +1768,8 @@ sui.views.editAttributeSelect2Field = sui.views.editAttributeField.extend( {
 
 		this.preselect( $field );
 
-		var $fieldSelect2 = $field.select2({
+                var select2 = jQuery().select2v4 ? 'select2v4' : 'select2';
```

Comment:
We could probably pass this handle in along with the localized script variables, so that we don't need to check which name it exists under here.